### PR TITLE
PhpUnitDedicateAssertFixer - Sort map and array by function name

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] sorts a map of functions and function names by function name

### CODE REVIEW

File: src/Fixer/PhpUnit/PhpUnitDedicateAssertFixer.php [hunk 1]
```diff
@@ -137,18 +137,18 @@ protected function createConfigurationDefinition()
...
             'is_boolean',
             'is_callable',
             'is_double',
             'is_float',
+            'is_infinite',
             'is_int',
             'is_integer',
             'is_long',
+            'is_nan',
+            'is_null',
```
Comment (keradus) [inline src/Fixer/PhpUnit/PhpUnitDedicateAssertFixer.php]: what about calling sort on it, to actually ensure values in readme are sorted, even if this list would not be ?

Comment (julienfalque) [reply src/Fixer/PhpUnit/PhpUnitDedicateAssertFixer.php]: Can't we replace this hardcoded array with `array_keys(self::$fixMap)`?

Comment (keradus) [reply src/Fixer/PhpUnit/PhpUnitDedicateAssertFixer.php]: no, next time someone will update possible options, he will not-willingly update default value as well

**Comment by SpacePossum** [comment]
@keradus can you kick circleci again, I don't have the permission to do so :|
@localheinz can you rebase this PR, sorry to bug you about it?

**Comment by localheinz** [comment]
@SpacePossum 

Rebased!

**Comment by SpacePossum** [comment]
thanks @localheinz :)

**Comment by keradus** [comment]
Thank you @localheinz.

**Comment by localheinz** [comment]
Thank you, @julienfalque, @keradus and @SpacePossum!

**Pull request closed without merging**